Python/tigre/algorithms/art_family_algorithms.py

```diff
@@ -98,10 +98,16 @@ def run_main_iter(self):
                     toc = default_timer()
                 if i == 1:
                     tic = default_timer()
+
+                    # Esitmated time until completetion in HH:MM:SS.ms
+                    remaining_time = ((self.niter - 1) * (tic - toc))
+                    seconds = int(remaining_time)
+                    milliseconds = str(remaining_time - seconds)[2:5]
                     print(
-                        "Esitmated time until completetion (s): "
-                        + str((self.niter - 1) * (tic - toc))
+                        "Esitmated time until completetion : "

```

Comment:
Typo, as explained in the other PR.  Same in all the others.

Reply:
`**"Esitmated time until completetion in HH:MM:SS.ms : "**` like this sir ?

Reply:
No.... check the words in English, they are wrong, they have typographical mistakes.

Reply:
completetion-> completion

Reply:
Sir, I have amend the changes as said by you.